config.yaml

```diff
@@ -3,45 +3,93 @@ options:
     default: True
     type: boolean
     description: |
-      Limit requested identifiers (hostnames, IPs and OIDs) to first requester
+      This filter is disabled by default, and can be enabled by setting the following config:
```

Comment:
"disabled by default" does not seem to match with the default config value.

Reply:
switched the description's places.